classes/publication/PKPPublication.php

```diff
@@ -70,72 +70,100 @@ public function getLocalizedData($key, $preferredLocale = null, &$selectedLocale
     /**
      * Combine the localized title, prefix and subtitle
      *
-     * @param string $preferredLocale Override the publication's default
-     *  locale and return the title in a specified locale.
-     *
+     * @param  string $preferredLocale  Override the publication's default locale and return the title in a specified locale.
+     * @param  string $format           Define the return data format as text or html
+     * 
      * @return string
      */
-    public function getLocalizedFullTitle($preferredLocale = null)
+    public function getLocalizedFullTitle($preferredLocale = null, string $format = 'text')
     {
-        $fullTitle = $this->getLocalizedTitle($preferredLocale);
-        $subtitle = $this->getLocalizedData('subtitle', $preferredLocale);
+        $fullTitle = $this->getLocalizedTitle($preferredLocale, $format);
+        $subtitle = $this->getLocalizedSubTitle($preferredLocale, $format);
```

Comment:
I think I prefer to keep the getters as "dumb" as possible, and only enrich as few of them as possible. (And in fact we're moving away from getters as much as possible; see the increased use of e.g. `$publication->getData('authors')` rather than `$submission->getAuthors()`.)

We need to have a "smart" accessor for getLocalizedFullTitle, because it already needs to combine several fields together, and now some of them will be HTML and not others. So rather than adding a `getLocalizedSubTitle` and passing `$format` down to it, I'd suggest only making `getLocalizedFullTitle` handle the logic.

Reply:
@asmecher I added the new method `getLocalizedSubTitle` as we need to pull the only the `sub title` differently in just few places(one example is crossref plugin) and using the `->getData`, we need to then apply `strip_tags` again .